### DISCUSSION TITLE
I cant seem to get MALT working!! I have it showing up on blender when you choose your render engine, but when I open up the node

### DISCUSSION THREAD

**Discussion by kiddkarma19** [bnpr/Malt#479] [Q&A]
I have it MALT up on blender when you choose your render engine, but when I open up the node tree window, it keeps saying that I have no nodes? what do I do?

**Comment by Wavenwater** [comment]
I have the same problem, I'd love to know how to get started with malt!
I'll sty tuned to figure it out as well!

![image](https://user-images.githubusercontent.com/120407433/217294408-e8f67a8a-0e0d-4311-9840-f21489a707d3.png)

**Comment by eballeste** [comment]
hey, have you taken a look at the [documentation](https://malt3d.com/Documentation/Getting%20Started/)?

**Reply by Wavenwater** [reply]
Yes I have taken a look at the documentation and I'm able to load the sample files and get a render, but I can't find or create any nodes in a new scene. The setup to make these nodes or how to find is also not documented in the page you sent me :s

**Reply by Wavenwater** [reply]
Also looked at this video, but can't do what the guys does at 1:18
https://www.youtube.com/watch?v=tE99jgCCcNE&t=705s&ab_channel=MiguelPozo
I don't know how to acces that Add pannel, I tried Shift+A, but it's not showing anything in a new scene...

**Comment by pragma37** [comment]
You have to select a node tree type before being able to add nodes:
![imagen](https://user-images.githubusercontent.com/36610999/217314081-b6385f55-b9bf-4ebe-9bf0-2167a472ec82.png)

We used to have a warning message for when a user tries to add a node without a tree type selected, but it looks like that's not working at the moment (I'll look into it ASAP).

**Reply by Wavenwater** [reply]
Oh wow, that is handy!
It works :D!

Thanks a lot friend!

**Reply by Wavenwater** [reply]
Now I have to figure out how to make this render and work by itself, I'll keep on looking out for this, thanks agaiN!
